test/studies/shootout/submitted/clbg-diff.py

```diff
@@ -70,17 +68,37 @@ def main():
         for s in submitted_missing:
             print(s)
 
+    print()
+    print('-------------------------------')
+    print('Checking for disabled shootouts')
+    print('-------------------------------')
+    print()
+    notested = glob.glob('*.notest')
+    futurized = glob.glob('*.future')
+    if notested or futurized:
+        print('Disabled tests found:')
+        for t in notested:
+            print(t)
+        for t in futurized:
+            print(t)
+
     print()
     print('----')
     print('Done')
     print('----')
     print()
 
 
-def cvs_checkout(r, d):
-    """Checkout cvs repository"""
-    cmd = 'cvs -d {0} checkout {1}'.format(r, d)
-    stdout, stderr = run_command(cmd, True)
+def git_checkout():
+    """Grab git source tarball"""
+    url = 'https://salsa.debian.org/benchmarksgame-team/benchmarksgame/raw/master/public/download/'
+    file = 'benchmarksgame-sourcecode.zip'
+    destdir = 'benchmarksgame'
+    stdout, stderr = run_command('rm -f ' + file, True)
+    stdout, stderr = run_command('rm -r ' + destdir, True)
+    stdout, stderr = run_command('wget '+ url + file, True)
+    stdout, stderr = run_command('unzip -d'+ destdir + ' ' + file, True)
+    stdout, stderr = run_command('rm ' + file, True)
```

Comment:
Same thing for `-f` here.

Reply:
In this case, the zipfile is guaranteed to exist if the previous commands were successful, though, right?

Reply:
Yes, but if the `wget` fails, then this generates an additional error. Not a big deal though (especially for a minor developer script).

Reply:
Oh, I was under the impression that the script failed in the presence of errors and figured it would never get this far (at least, that's what I was seeing when I had other commands failing yesterday...  but I'll admit I don't know the first thing about Python failure modes or behavior).